src/Database/Apprenticeship.sql

```diff
@@ -1,21 +1,10 @@
 ﻿CREATE TABLE [dbo].[Apprenticeship]
 (
 	[Key] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
+    [ApprovalsApprenticeshipId] BIGINT NOT NULL, --TODO INFORM MIKE THAT THIS NEEDING TO BE IN DESIGN (
```

Comment:
Do we really need unicode characters for ULN (appreciate this is how it was before the change)